src/service/resources-impl.js

```diff
@@ -158,9 +158,15 @@ export class Resources {
     /** @private {number} */
     this.lastVelocity_ = 0;
 
-    /** @const {!Pass} */
+    /** @const @private {!Pass} */
     this.pass_ = new Pass(this.win, () => this.doPass());
 
+    /** @const @private {!Pass} */
+    this.remeasurePass_ = new Pass(this.win, () => {
+      this.relayoutAll_ = true;
+      this.schedulePass();
+    }, 1000);
```

Comment:
Let's change this to `0` since you are calling below `schedulePass(1000)` and to avoid adding an extra second in the font timeout code.